apps/website/src/app/page.tsx

```diff
@@ -27,7 +27,8 @@ export default function Home() {
                         alt="Midnight whispers image"
                         src="https://semaphore.cedoor.dev/midnight-whispers.jpg"
                         objectFit="cover"
-                        w="full"
+                        minWidth="100%"
```

Comment:
We could need this. I've tried it with `overflowX: "hidden"` only and it works. Could you double-check?

Reply:
You mean, the `w="full"`?

Reply:
Sorry, * we could not need this.

`overflowX: "hidden"` could be enough for making it work.

Reply:
No, if you remove the parameter `minWidth="100%"` the images do not extend horizontally and you get something like this

![image](https://github.com/semaphore-protocol/semaphore/assets/20580910/d0f690d0-1c09-460c-a193-edd551fafba5)

Reply:
Umh I'm trying it with `w="full"` and it works

Reply:
Yes, maintaining `w="full"` works since is equivalent to min & max at 100%. Lmk which ones do you prefer

Reply:
Better for consistency across images, see last commit